src/sdltiles.cpp

```diff
@@ -98,124 +99,11 @@ static uint32_t lastupdate = 0;
 static uint32_t interval = 25;
 static bool needupdate = false;
 
-std::array<SDL_Color, color_loader<SDL_Color>::COLOR_NAMES_COUNT> windowsPalette;
```

Comment:
Some unused code above should be deleted 

![image](https://user-images.githubusercontent.com/16213433/90428955-768b9c80-e0cd-11ea-8e26-11820cb70312.png)